### PR TITLE
Update Chrome driver

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: src/Shared/E2ETesting/selenium-config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "drivers": {
     "chrome": {
-      "version" : "91.0.4472.77"
+      "version" : "91.0.4472.19"
```
Comment (JunTaoLuo) [inline src/Shared/E2ETesting/selenium-config.json:4]: Usually we try to match this version and the SeleniumWebDriverChromeDriverVersion: https://www.nuget.org/packages/Selenium.WebDriver.ChromeDriver/91.0.4472.1900-beta at https://github.com/dotnet/aspnetcore/blob/main/eng/Versions.props#L250.

Comment (BrennanConroy) [reply src/Shared/E2ETesting/selenium-config.json:4]: It's non-obvious what the relationship is between those two things, and I'd like to unblock builds

Comment (dougbu) [reply src/Shared/E2ETesting/selenium-config.json:4]: The relationship is mainly to the version of Chrome installed. Kestrel's Interop.FunctionalTests.csproj _should_ fail w/ the current version or will real soon now. Plz fix if possible though separate PR may be fine.

**Review by dougbu — APPROVED** [review]
For the builds&hellip;

File: src/Shared/E2ETesting/selenium-config.json (hunk 1 above)
Comment (BrennanConroy) [reply src/Shared/E2ETesting/selenium-config.json:4]: Someone will need to add the package to the dotnet-public feed

**Comment by dougbu** [comment]
Well, that didn't work and I'm working on other manual backports. Can you handle this one @BrennanConroy❔

**Pull request merged**